pkg/executor/core/executor.go

```diff
@@ -118,6 +119,10 @@ func (executor Executor) verifySubjectInternal(ctx context.Context, verifyParame
 
 	overallVerifySuccess := executor.PolicyEnforcer.OverallVerifyResult(ctx, verifierReports)
 
+	if executor.Config.ExecutionMode == config.PassthroughExecutionMode {
```

Comment:
suggestion to add a log message like "Passthrough execution mode, policy configuration ignored ."